L1Trigger/L1TMuonEndCap/python/simEmtfDigis_cfi.py

```diff
@@ -164,4 +164,4 @@
 
 ## Era: Run3_2021
 from Configuration.Eras.Modifier_stage2L1Trigger_2021_cff import stage2L1Trigger_2021
-stage2L1Trigger_2021.toModify(simEmtfDigis, RPCEnable = cms.bool(True), UseRun3CCLUT_OTMB = cms.bool(False), Era = cms.string('Run3_2021'))
+stage2L1Trigger_2021.toModify(simEmtfDigis, RPCEnable = cms.bool(True), UseRun3CCLUT_OTMB = cms.bool(True), Era = cms.string('Run3_2021'))
```

Comment:
```suggestion
stage2L1Trigger_2021.toModify(simEmtfDigis, RPCEnable =True, UseRun3CCLUT_OTMB = True, Era = 'Run3_2021')
```
and please do the same for all other modifications here above